tests/test_deprecations.py

```diff
@@ -295,3 +296,79 @@ def test_deprecations_freq_range_parameter_renaming_results():
         pf.dsp.filter.fractional_octave_bands(sweep, 8,
                                               frequency_range=(20, 20e3)).time,
         rtol=0)
+
+
+def test_deprecation_shelve_functions():
+    # test high_shelve()
+    with pytest.warns(
+        PyfarDeprecationWarning, match="'high_shelve' will be deprecated in "
+            "pyfar 0.9.0 in favor of 'high_shelf'"):
+        pfilt.high_shelve(None, 10e3, 1.5, 1, "I", sampling_rate=44100)
+
+    if version.parse(pf.__version__) >= version.parse('0.8.0'):
```

Comment:
Should be changed here and below to:
`if version.parse(pf.__version__) >= version.parse('0.9.0'):`